int32 roomId
string roomName
int32 batteryLevel